aether/sdk/conf/settings.py

```diff
@@ -551,6 +551,11 @@ def get_required(name):
             os.environ.get('MINIO_STORAGE_MEDIA_BACKUP_BUCKET')
         )
 
+        if bool(os.environ.get('COLLECT_STATIC_FILES_ON_STORAGE')):
+            STATICFILES_STORAGE = 'minio_storage.storage.MinioMediaStorage'
+            MINIO_STORAGE_STATIC_BUCKET_NAME = f'{get_required("BUCKET_NAME")}-static'
+            MINIO_STORAGE_AUTO_CREATE_STATIC_BUCKET = True
```

Comment:
This should come from an environment variable, we cannot assume that the minio user is granted to create buckets.